### PR TITLE
WIP add forgot password page

### PR DESCRIPTION
**Pull request by idanre**
still a WIP for now
opened the pull request for testing and initial feedback

### CODE REVIEW

**Comment by guyluz11** [comment]
#134

**Comment by guyluz11** [comment]
Thanks Idanre looks good.

Can you add translation support for all the text?.
For that to happen you need to import `import 'package:easy_localization/easy_localization.dart';`
 add `.tr()` to the end of the `Text` widget, after that change the text spaces to underscore (`_`) and add it to the file `en-US.json`.
The left side should have the text with the underscore and the right side should have the actual text that you want to show up.

And please add documentation to the `class ForgotPasswordPage` with `///` text above it.
Thanks

File: lib/features/forgot_password/forgot_password_page.dart [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    final Hero logo = Hero(
+      tag: 'hero',
+      child: CircleAvatar(
+        backgroundColor: Colors.transparent,
+        radius: 48.0,
+        child: Image.asset('assets/logo.png'),
+      ),
+    );
+
+    const Text title = Text('Forgot Password',
```
Comment (guyluz11) [inline lib/features/forgot_password/forgot_password_page.dart:18]: Missing translation support.

File: lib/features/forgot_password/forgot_password_page.dart [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import 'package:flutter/material.dart';
+import 'package:easy_localization/easy_localization.dart';
+
```
Comment (guyluz11) [inline lib/features/forgot_password/forgot_password_page.dart:3]: Add class documentation

File: lib/features/forgot_password/forgot_password_page.dart [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    final Padding button = Padding(
+      padding: const EdgeInsets.symmetric(vertical: 16.0),
+      child: RaisedButton(
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(24),
+        ),
+        onPressed: ()=> {},
+        padding: const EdgeInsets.all(12),
+        color: Theme.of(context).accentColor,
+        child: const Text('Submit', style: TextStyle(color: Colors.white)),
```
Comment (guyluz11) [inline lib/features/forgot_password/forgot_password_page.dart:43]: Missing translation support.

**Review by guyluz11 — COMMENTED** [review]
Please fix all the part.
If you want you can add the validation for the Email field, you can use the code and the regex from `form_widget.dart`

File: lib/features/forgot_password/forgot_password_page.dart (hunk 2 above)
Comment (idanre) [reply lib/features/forgot_password/forgot_password_page.dart:3]: example? i don't see anything in other classes of this project.

Comment (guyluz11) [reply lib/features/forgot_password/forgot_password_page.dart:4]: in `add_new_cbj_devices.dart` class `AddNewCBJDevices` you can see one line above it `/// hotspot of the app device.`

**Comment by guyluz11** [comment]
Thanks for the pr

**Pull request merged**